enhancements/update/upgrade-without-registry.md

```diff
@@ -0,0 +1,989 @@
+match it will report it as an error condition in the `ClusterVersion` object and
+the upgrade process will be aborted.
+
+If there is enough space the `bundle-extractor` will extract the contents of
+the bundle to the `/var/lib/upgrade/4.13.7-x86_64` directory.
+
+Once the digest has been verified and the bundle has been completely extracted
+the `bundle-extractor` will update the status of the `ClusterVersion` object to
+indicate that the bundle is extracted in that node. This will be done via a new
+`status.desired.bundle.nodes.*.extracted` and
+`status.seried.bundle.nodes.*.metadata` fields. For example, when `node0` and
+`node2` have completed the extraction of the bundle but `node1` hasn't, the
+`ClusterVersion` status will look like this:
+
+```yaml
+status:
+  desired:
+    bundle:
+      monitor: true
+      file: /dev/sdb
+      nodes:
+        node0:
+          extracted: true
+          metadata: |
+            {
+              "version": "4.13.7",
+              "arch": "x86_64",
+              "release": "quay.io/openshift-release-dev/ocp-release@sha256:...",
+              "images": [
+                "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:...",
+                "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:...",
+                ...
+              ]
+            }
+        node1:
+          extracted: false
+        node2:
+          extracted: true
+          metadata: |
+            {
+              "version": "4.13.7",
+              "arch": "x86_64",
+              "size": "16 GiB",
+              "release": "quay.io/openshift-release-dev/ocp-release@sha256:...",
+              "images": [
+                "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:...",
+                "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:...",
+                ...
+              ]
+            }
+```
+
+At that point the `bundle-extractor` job will finish.
+
+The `extracted` field for each node will indicate if the bundle extraction
+process has been completed.
+
+The `metadata` field for each node will contain the information from the
+`metadata.json` file of the bundle. Note that the list of images may be too long
+(approximately 180 images) to store it in the status of the `ClusterVersion`
+object; it may be convenient to store them in separate configmaps, and have only
+the references to those configmaps in the `ClusterVersion` status.
+
+When all the nodes have the bundle extracted (the `extracted` field for all
+nodes is `true`) the cluster version operator will delete the `bundle-server`
+daemon set and verify that the metadata in all nodes (the content of the
+`metadata` field) is the same. If there are differences they will be reported as
+error conditions in the status of the `ClusterVersion` object and the upgrade
+process will be aborted. This is intended to prevent accidents like having two
+different USB sticks with different bundles plugged in two different nodes.
```

Comment:
Yes, but that checksum is optional.

Reply:
Why provide multiple solutions to the same problem?

Reply:
Right, so, the intention is that if the user hasn't set the digest deliberately, each node checks in with the digest of the bundle it has, and, if any of those differ, the entire cluster upgrade stops?

It would be worth providing an example of how the user would observe this error and from that how they would understand the fixes required